becquerel/parsers/spectrum_file.py

```diff
@@ -56,8 +56,7 @@ def __init__(self, filename):
         self.cal_coeff = []
         # arrays to be calculated using calibration
         self.energies = np.array([], dtype=np.float)
-        self.energy_bin_widths = np.array([], dtype=np.float)
```

Comment:
Don't remove. You can change this to `bin_widths_keV`, create an alias property for `energy_bin_widths` and have the `bin_widths_keV` property raise a `DeprecationWarning`